actioncable/lib/action_cable/connection/base.rb

```diff
@@ -126,7 +126,8 @@ def on_message(message) # :nodoc:
       end
 
       def on_error(message) # :nodoc:
-        # ignore
+        # log errors to make diagnosing socket errors easier
+        logger.error "Error writing to Action Cable socket: #{message}"
```

Comment:
Awesome. It's changed. Will this be all, then I can squash the commit and it can be merged in :)